### ISSUE TITLE
"Can't execute code from a freed script" error when testing Sencha ExtJS applications in IE11

### ISSUE THREAD

**Issue by kot-lex** [DevExpress/testcafe#2639]
### Are you requesting a feature or reporting a bug?
Bug

### What is the current behavior?
Test of ExtJS application in IE11 fails with "Can't execute code from a freed script" 

### What is the expected behavior?

Test passes as it does in all other browsers

### How would you reproduce the current behavior (if this is a bug)?

Clone repo with example https://github.com/kot-lex/testcafe-test
run `npm install`
run `npm run test`

#### Provide the test code and the tested page URL (if applicable)

Repo with sample test https://github.com/kot-lex/testcafe-test

Tested page URL:
http://examples.sencha.com/extjs/6.6.0/examples/admin-dashboard/

Test code

```js
test('My first test', async t => {
    const button = Selector('.x-treelist-item-text').withExactText('Email');
    await t
        .click(button)
    });
```

### Specify your

* operating system: Windows 10
* testcafe version: 0.20.4
* node.js version: 8.11.3

**Comment by AndreyBelym** [comment]
Hi @kot-lex, I've reproduced the issue. We need some time to investigate what's going wrong on the page.

**Comment by kot-lex** [comment]
Hi @AndreyBelym, any update on this?

**Comment by kot-lex** [comment]
Any news? Ability to run test in Internet Explorer 11 is critical for us.

**Comment by AndreyBelym** [comment]
Hi @kot-lex, unfortunately page's code is very complex, and we don't have any significant results.

**Comment by kot-lex** [comment]
@AndreyBelym is there any chance of this issue being resolved by your team anytime soon? 
If there is anything we can do on our side to help you address the issue please let me know.

**Comment by Farfurix** [comment]
I'm working on it.

**Comment by miherlosev** [comment]
Fixed in https://github.com/DevExpress/testcafe/pull/2779

**Comment by AndreyBelym** [comment]
Hi @kot-lex! The fix is released in `testcafe@0.22.0`, you can install it with `npm i testcafe@latest` and try it on your machine.

**Comment by kot-lex** [comment]
Works like a charm, thank you guys!
